apps/timetable/admin/index.html

```diff
@@ -140,11 +140,23 @@
             <%= _.partial('new-module-modal', {'data': data}) %>
         </script>
 
+        <!-- Rename module modal -->
+        <div id="gh-rename-module-modal-container"><!-- --></div>
+        <script id="gh-rename-module-modal-template" type="text/template">
+            <%= _.partial('rename-module-modal', {'data': data}) %>
+        </script>
+
         <!-- Batch edit template -->
         <script id="gh-batch-edit-template" type="text/template">
             <%= _.partial('admin-batch-edit', {'data': data}) %>
         </script>
 
+        <!-- Delete module modal -->
+        <div id="gh-delete-module-modal-container"><!-- --></div>
+        <script id="gh-delete-module-modal-template" type="text/template">
+            <%= _.partial('delete-module-modal', {'data': data}) %>
```

Comment:
The modal partial contains a partial for the delete overview which needs data to work. Therefor we need to pass in an empty data object here so that the partial doesn't error out. The partial won't actually be rendered inside of the modal yet because of the `false` flag but it will need at least an empty data object or fail. I can elaborate offline if it's not clear.